DEVELOPMENT.md

```diff
@@ -64,7 +64,6 @@ You now need to configure the system so that it can find various bits and pieces
 ,   "hookPath":         "api/hook"
     // if you aren't running ash-nazg at the root of a site, you can define a prefix so that it
     // knows where its actual root is. Otherwise /.
-,   "urlPathPrefix":    "/"
```

Comment:
@tripu I see you removed the setting option but left the comment. Is that intended ?

Reply:
:+1:

Reply:
Ah, I didn't notice the comment. Thanks!